### DISCUSSION TITLE
Remotely allows for unregistered users to register

### DISCUSSION THREAD

**Discussion by Baltimorepc** [immense/Remotely#746] [Q&A]
when you get to the remotely page. it allows a user to create a account and then able to us it as there own organization server and makes them an admin of the one they created and i cant even delete them how can i fix this. As i tested it by creating a user without sending an invite and it sent the conformation email and then brought me to my own organization home page. i dont want others to be able to use this as there own without my approval. How can i fix this any help would be great thank you

**Comment by Baltimorepc** [comment]
it seems this is the issue even thought i registered my server some reason it doesnt register it like gitup says and then removes it An organization is automatically created for the account.
Organizations are used to group together users, devices, and other data items into a single pool.
By default, only one organization can exist on a server.
The Register button will disappear.
People will no longer be able to create accounts on their own.

since people can still register on there own hopefully they fix this in the next update ALSO the End User Support Page isnt working its saying device ID is missing

**Comment by Baltimorepc** [comment]
Can anyone help with this. Can i remove the Register as new user option,  As i dont want no one registering to use my server without my permission. Or creating another Organization but me. they can download the remote without logging in. i only wanna be able to send out invites. But right now i tested and created a new account and it will allow anyone to register. Or am i seeing or understand

**Reply by BJSchmidt** [reply]
You probably changed the Max Organization Count from 1 to -1 (which allows unlimited organizations to be created.) Set the Max Organization Count back 1, and no more organizations will be able to be created, so the Register option is removed.
